src/utils/fishData.js

```diff
@@ -184,7 +187,8 @@ const fishComponents = {
       dorsalFinAnchor: [76, -13],
       tailAnchor: [144, 56],
       knnData: [12419, BodyShape.TRIANGLE],
-      type: FishBodyPart.BODY
+      type: FishBodyPart.BODY,
+      exclusions: [DataSet.Small]
```

Comment:
So if a dataset is listed here, this component _isn't_ included in that set?

Reply:
yeah. it feels a little backwards, but it made more sense to me to have components opt-out rather than have to remember to opt-in